App/Redux/ProcessingImagesRedux.ts

```diff
@@ -54,24 +54,24 @@ export type ProcessingImagesAction = ActionType<typeof actions>
 
 export type ProcessingImage = {
   readonly uuid: string,
-  readonly sharedImage: SharedImage
-  readonly destinationThreadId: string
+  readonly sharedImage: TT.SharedImage
+  readonly destinationThreadId: TT.ThreadId
```

Comment:
Got sick of typing out TextileTypes all over, so standardized to TT

Reply:
See my comment in `TextilePhotos`.